LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2023 Red Hat Inc.
+   Copyright 2024 Red Hat Inc.
```

Comment:
```suggestion
   Copyright Red Hat
```

I just found an article titled `Copyright Notices for Open Source Projects` from the Legal team on The Source. It clearly does not make it mandatory to include the year:

> [...]
> We do not recommend inclusion of the year in the copyright notice. Attempting to consistently maintain dates in copyright notices is difficult, time-consuming, and is unnecessary.

So I think we can just simplify and omit the year as well.

It also recommends "Red Hat" over "Red Hat, Inc."